cli/tests/integration_test.rs

```diff
@@ -25,12 +25,48 @@ use anyhow::Result;
 use assert_cmd::Command;
 
 #[test]
-fn should_output_help() -> Result<()> {
+fn outputs_help() -> Result<()> {
     env!("CARGO_BIN_EXE_cli");
     let assertion = Command::cargo_bin("cli")?.arg("--help").assert();
     let bytes = assertion.get_output().stdout.as_slice();
     let output = str::from_utf8(bytes)?;
-    assert!(output.contains("The port to run"));
-    assert!(output.contains("3000"));
+    assert!(output.contains("help"));
+    assert!(output.contains("start"));
+    assert!(output.contains("Start the server"));
+    assert!(output.contains("--help"));
+    assert!(output.contains("Print help"));
+    Ok(())
+}
+
+#[test]
+fn outputs_start_opts() -> Result<()> {
+    env!("CARGO_BIN_EXE_cli");
```

Comment:
@h1alexbel btw I'm not sure that this is an required, can you try to remove it?

Reply:
@l3r8yJ removed, test passed.